Source/WebCore/testing/Internals.cpp

```diff
@@ -2351,7 +2351,7 @@ void Internals::advanceToNextMisspelling()
 
 Vector<String> Internals::userPreferredLanguages() const
 {
-    return WTF::userPreferredLanguages();
+    return WTF::userPreferredLanguages(ShouldMinimizeLanguages::No);
```

Comment:
Ditto again.

Reply:
This one I think I should do in this patch. This is needed by a lot of tests and this is test-only. No fingerprinting risk or risk or breakage.

Reply:
🆗